Answers.md

```diff
@@ -1 +1,13 @@
-
+1. Describe the biggest difference between `.forEach` & `.map`.
+    .for each does not return anything but .map does return
```

Comment:
The biggest takeaway from that should be:
forEach manipulates the array while map makes a copy.
there will be instances where you don't want to mess with the original